android-start/app/build.gradle

```diff
@@ -45,12 +46,17 @@ dependencies {
     compile 'com.google.android.gms:play-services-appinvite:11.4.2'
     compile 'com.google.firebase:firebase-messaging:11.4.2'
     compile 'com.google.android.gms:play-services-ads:11.4.2'
-    compile 'com.google.firebase:firebase-crash:11.4.2'
     compile 'com.google.firebase:firebase-appindexing:11.4.2'
 
     // Firebase UI
     compile 'com.firebaseui:firebase-ui-database:3.0.0'
 
+    //Crashlytics
+    compile('com.crashlytics.sdk.android:crashlytics:2.7.1@aar') {
```

Comment:
Just curious, would this whole `compile() { ... }` block be equivalent as:

```groovy
// Drop @aar and the transitive option (which is true by default)
compile 'com.crashlytics.sdk.android:crashlytics:2.7.1`
```

Reply:
Looking at the POM:
https://maven.fabric.io/public/com/crashlytics/sdk/android/crashlytics/2.7.1/crashlytics-2.7.1.pom

Seems like it would be, given that `<packaging>aar</packaging>` and all the `dependencies` are specified correctly.